src/EnergyPlus/CMakeLists.txt

```diff
@@ -262,6 +262,8 @@ set(SRC
     EvaporativeFluidCoolers.hh
     ExhaustAirSystemManager.cc
     ExhaustAirSystemManager.hh
+    extendedHI.cc
+    extendedHI.hh
```

Comment:
I don't like the lower cased names...  I mean in general I am fine with it, but it looks super weird against all the other PascalCased names.  I'm going to change this if I push changes up.

Reply:
Thanks. I just changed the filenames to PascalCased names.